flow/connectors/postgres/qrep_query_executor.go

```diff
@@ -115,10 +116,21 @@ func (qe *QRepQueryExecutor) fieldDescriptionsToSchema(fds []pgconn.FieldDescrip
 		// there isn't a way to know if a column is nullable or not
 		// TODO fix this.
 		cnullable := true
-		qfields[i] = model.QField{
-			Name:     cname,
-			Type:     ctype,
-			Nullable: cnullable,
+		if ctype == qvalue.QValueKindNumeric {
+			precision, scale := numeric.ParseNumericTypmod(fd.TypeModifier)
+			qfields[i] = model.QField{
+				Name:      cname,
+				Type:      ctype,
+				Nullable:  cnullable,
+				Precision: int64(precision),
+				Scale:     int64(scale),
```

Comment:
```suggestion
				Precision: int64(precision),
				Scale:     int64(scale),
```
int64 seems a bit much, these are 16 bit values in postgres iirc